block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go

```diff
@@ -144,6 +155,69 @@ var _ = framework.KubeDescribe("[Feature:Block_Volume_Attach_E2E]", func() {
 				logResult("BlockVolumeAttacher-Volume-Test: Attach: PASS\n")
 			}
 
+			/* Port Worx PVC creation */
+			portworxpvcpath = gopath + "/" + portworxpvcpath
+			portworxscpath = gopath + "/" + portworxscpath
+			filestatus, err = fileExists(portworxpvcpath)
+			if err != nil {
+				cleanUP(pvname, pv)
+				logResult("BlockVolumeAttacher-Volume-Test: PVC Creaiton: FAIL\n")
+			} else {
+				logResult("BlockVolumeAttacher-Volume-Test: PVC Creaiton: PASS\n")
+			}
+			Expect(err).NotTo(HaveOccurred())
+
+			/* PVC Creation */
+
+			By("PVC  Creation")
+			if filestatus == true {
+				filepatharg := fmt.Sprintf("%s", portworxpvcpath)
+				filepatharg2 := fmt.Sprintf("%s", portworxscpath)
+				cmd := exec.Command(pvscriptpath, filepatharg, "portworxpvcreate", filepatharg2)
```

Comment:
I dont see `filepatharg` is being used. Is this param needed to be passed?

Reply:
Removed